source/nvwave.py

```diff
@@ -99,9 +100,9 @@ class AudioPurpose(Enum):
 	SOUNDS = auto()
 
 
-def _getOutputDevices():
-	"""Generator, returning device ID and device Name in device ID order.
-	@note: Depending on number of devices being fetched, this may take some time (~3ms)
+def _getOutputDevices() -> Iterator[tuple[str, str]]:
```

Comment:
I'd suggest using the Generator type as their handling can be different

```suggestion
def _getOutputDevices() -> Generator[tuple[str, str], None, None]:
```